ci-operator/step-registry/ignition/bastionhost/ignition-bastionhost-commands.sh

```diff
@@ -232,7 +236,7 @@ cat > "${rsyncd_ignition_patch}" << EOF
         "path": "/etc/rsyncd.conf",
         "overwrite": true,
         "contents": {
-          "source": "data:text/plain;base64,${RSYNCD_CONFIG_CONTENT}"
+          "source": "${RSYNCD_CONFIG_CONTENT}"
```

Comment:
Why removed `data:text/plain;base64,` here ?

Reply:
I just run a [job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/65506/rehearse-65506-periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-nightly-azure-ipi-private-f28-disasterrecovery/1930204388450635776) against azure , the change seem breaking the bastion provision.

Reply:
From IBMCloud support, this issue is caused by Cloudflare, I have to change the RSYNCD_CONFIG_CONTENT in IBMCloud